JSDemos/Demos/TextArea/Overview/description.md

```diff
@@ -1 +1,7 @@
-The TextArea is&nbsp;a&nbsp;UI component that enables a&nbsp;user to&nbsp;enter and edit a&nbsp;multi-line text.
\ No newline at end of file
+The TextArea enables users to enter and edit a multi-line text. The component can have fixed or resizable [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height). The component with the fixed **height** displays a scroll bar if the entered text exceeds the physical space of the text area. If you set the [autoResizeEnabled](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#autoResizeEnabled) property to **true**, the TextArea automatically resizes its height to fit the text.
+
+You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to deny users to enter more than allowed number of characters. Note that this property only limits the number of characters for users, you can still set the text of the desired length programmatically.
```

Comment:
1. Can we update the first sentence as follows?

`You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters a user can enter.`

2. `Note that this property only limits the number of characters for users, you can still set the text of the desired length programmatically.`

I think we can separate this sentence into two sentences. I'd also like to update the second part of the sentence. What do you think about the following?

`Note that this property only limits the number of characters for users. You can exceed the maximum character length if you set the text programmatically.`

Reply:
1. Okay, let's update this sentence.
2. I mean that if you set the maxLength property, for example, to 100 characters, the original text that is stored in the value property can still be over 100 characters (it can be as long as the programmer wants). The users will see the first 100 characters of the text in this situation. I think your second suggestion is more suitable, but I still think it's a bit off.

Reply:
Thanks for the clarification. What do you think about the following?

`You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters a user can enter.

Note that this property only limits the number of characters for users. You can enter text that exceeds the maximum character length programmatically, but the number of characters displayed to users will still be limited by this property setting.`

Let me know what you think.

Reply:
I changed it. Thanks for your suggestion.

Reply:
Thank you.